DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv4/conftest.py

```diff
@@ -0,0 +1,37 @@
+import pytest_asyncio
+
+from dent_os_testbed.lib.os.sysctl import Sysctl
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+)
+
+
+@pytest_asyncio.fixture(autouse=True)
```

Comment:
Maybe even make it scope=module?

Reply:
Using `scope=module` will throw a `ScopeMismatch` exception. I did not look into how it can be fixed.